build/conda_builder/aarch64/Dockerfile

```diff
@@ -35,7 +36,7 @@ WORKDIR /home/saluser
 
 ARG PYTHON_VERSION=3.11
```

Comment:
Does this matter at all? Didn't get switched to 3.12

Reply:
Technically no, it gets overwritten by the value provided by cycle.env.